src/quick_start/browser_extension.md

```diff
@@ -67,10 +67,15 @@ If you plan to run a local notary server:
    ```
 2. Edit the notary server config file (`notary-server/config/config.yaml`) to turn off TLS so that the browser extension can connect to the local notary server without requiring extra steps to accept self-signed certificates in the browser.
    ```yaml
-   tls-signature:
-      enabled: false
+    tls:
+        enabled: false
+        ...
    ```
-3. Run the notary server:
+3. Switch to version `v0.1.0-alpha.3`
```

Comment:
Added this because the extension download link written at the top of this file is referring to alpha.3